CHANGELOG.md

```diff
@@ -1,17 +1,47 @@
 # Changelog
-
-## 2.1.2
+All notable changes to this project will be documented in this file.
+
+The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.2.0] - 2018-05-14
+### Added
+- Rights to user set own User-Agent
+- Getter $api->getUserAgentHeader() for retrieving HTTP User-Agent header
+- Getter $api->getIntegrationData() for data-integration attribute to the widget
+- Setters $api->setFramework($name, $version), $api->setExtension($name, $version), $api->setUserAgentName($name) methods reported during API requests
+- Getters $api->getFramework(), $api->getExtension(), $api->getLibraryName(), $api->getUserAgentName()
+- @throws to PHPDoc
```

Comment:
lines 10-14 should not be here(this is not git log).
If you want to add this, rephrase and add as single item

e.g. changed: user agent reporting for the lib and integrations that use it

Reply:
Ok